pages/api/id/[form]/owners.ts

```diff
@@ -38,5 +40,40 @@ export async function getEmailListByFormID(
   //Could not find formID in the path
   return res.status(400).json({ error: "Malformed API Request FormID not define" });
 }
+/**
+ * It will activate and deactivate all the owners associated to a specific form.
+ * This function is expecting to find a payload in the request body like:
+ * { "email": "forms@cds.ca", "active":"0/1"}
+ * "email": The email of the user associated to the form
+ * "active": Boolean value indicating whether the form owner is active or not
+ * @param req
+ * @param res
+ */
+export async function activateOrDeactivateFormOwners(
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> {
+  //Extracting req body
+  const requestBody =
+    req.body && Object.keys(req.body).length > 0 ? JSON.parse(req.body) : undefined;
+  //Payload validation
+  if (!requestBody?.email || !requestBody?.active) {
+    //Invalid payload
+    return res.status(400).json({ error: "Invalid payload fields are not define" });
+  }
 
-export default isRequestAllowed(["GET", "POST"], isUserSessionExist(owners));
+  const formID = req.query.form as string;
+  const { email, active } = requestBody;
+  if (!formID) return res.status(400).json({ error: "Malformed API Request Invalid formID" });
+  //Update form_users's records
+  const resultObject = await executeQuery(
```

Comment:
How are you sure here that active is not a value of "sdbdbfwbfwe" for example. Why are we using string "1" or "0" instead of true or false

Reply:
Will support both format  1/0 and true/false. I've also added a test case and a validation point.

Reply:
sample payload format : 
`{       
        "email": "forms@cds.ca",
        "active":  "true"
}`

Reply:
Why is true or false a string here. It should just be a boolean true or false

Reply:
Yeah , this was just a sample. But you can find more examples in our tests.  Like `{ id: "2", email: "forms@cds.ca", active: false },`